src/base/storage.rs

```diff
@@ -196,8 +196,19 @@ pub unsafe trait RawStorageMut<T, R: Dim, C: Dim = U1>: RawStorage<T, R, C> {
     /// If the indices are out of bounds, the method will cause undefined behavior.
```

Comment:
Maybe something like this?
```suggestion
    /// If the indices are out of bounds, the method will cause undefined behavior.
    /// 
    /// # Validity
    /// The default implementation of this trait function is only guaranteed to be
    /// sound if invocations of `self.ptr_mut()` and `self.get_address_unchecked_linear_mut()`
    /// result in stable references. If any of the data pointed to by these trait methods
    /// moves as a consequence of invoking either of these methods then this default
    /// trait implementation may be invalid or unsound.
    
```